Sprint-2/errors/2.js

```diff
@@ -3,8 +3,15 @@
 
 // this function should square any number but instead we're going to get an error
 
-function square(3) {
+// function square(3) {
+//     return num * num;
+// }
+
+// correction 
+// the problem in the above code is it didnt defined the num as a parameter of square.
```

Comment:
Can 3 ever be a parameter in JavaScript? What kind of error did this code produce?